packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -19,6 +19,7 @@ import {
     SwapQuote,
     SwapQuoteRequestOpts,
     SwapQuoterOpts,
+    SwapQuoterRfqtOpts,
 } from './types';
 import { assert } from './utils/assert';
 import { calculateLiquidity } from './utils/calculate_liquidity';
```

Comment:
All the changes to this class are just to support the generation of a new `QuoteRequestor` for each request, instead of using one `QuoteRequestor` for every request.   This is because we now store state on the `QuoteRequestor`